src/lib/config.js

```diff
@@ -5,16 +5,16 @@ module.exports = {
     BASE_URL: process.env.API_BASE_URL || "http://localhost:5007/",
     PATHS: {
       AUTHORIZE: "session",
-      CHECK: "check",
+      CHECK: "identity-check",
     },
   },
   APP: {
-    BASE_URL: process.env.EXTERNAL_WEBSITE_HOST || "http://localhost:5040",
+    BASE_URL: process.env.EXTERNAL_WEBSITE_HOST || "http://localhost:5030",
```

Comment:
re: ports - I'd normally say, let the other team change their values, but it doesn't matter too much here :)

You'll also need to update the `local.Dockerfile`, and the `README.md`